<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerDynamicModelerModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerDynamicModelerModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="ButtonLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Create:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="qMRMLSubjectHierarchyTreeView" name="SubjectHierarchyTreeView">
     <property name="highlightReferencedItems">
      <bool>false</bool>
     </property>
     <property name="nodeTypes">
      <stringlist>
       <string>vtkMRMLDynamicModelerNode</string>
      </stringlist>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="InputNodesCollapsibleButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Input nodes</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3"/>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="ParametersCollapsibleButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Parameters</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4"/>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="OutputNodesCollapsibleButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Output nodes</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5"/>
    </widget>
   </item>
   <item>
    <widget class="ctkCheckablePushButton" name="ApplyButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="checkState">
      <enum>Qt::Unchecked</enum>
     </property>
     <property name="checkBoxControlsButtonToggleState">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLSubjectHierarchyTreeView</class>
   <extends>QTreeView</extends>
   <header>qMRMLSubjectHierarchyTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCheckablePushButton</class>
   <extends>ctkPushButton</extends>
   <header>ctkCheckablePushButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkPushButton</class>
   <extends>QPushButton</extends>
   <header>ctkPushButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerDynamicModelerModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>SubjectHierarchyTreeView</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
